### DISCUSSION TITLE
Mail Server

### DISCUSSION THREAD

**Discussion by M1st3r3** [requarks/wiki#6315] [Help / Questions] [answered]
Hi i just wanted to know , i set up the mail server in the options . When i click send test email it works perfectly i receive the mail .  But how can i set it up to be like when a user comment on the page u created u receive an email or when the user from the AD first connect it receive a welcome email ? Sorry English is not my first language

**Comment by NGPixel** [comment, marked as answer]
That's not supported yet.
